bot/exts/utilities/realpython.py

```diff
@@ -32,13 +31,21 @@ def __init__(self, bot: Bot):
 
     @commands.command(aliases=["rp"])
     @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
-    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str) -> None:
+    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str = None) -> None:
```

Comment:
If defaulting to None, would this not be an Optional[str]?

```suggestion
    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: Optional[str] = None) -> None:
```

Reply:
This is invalid syntax.

Reply:
```
File "/home/xithrius/Repositories/PythonDiscord/sir-lancebot/bot/exts/utilities/realpython.py", line 34
    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, Optional[user_search]: str = None) -> None:
                                                                                            ^
SyntaxError: invalid syntax
```

Reply:
Fixed in dc4bac651f71707b1e2144ac3b94e8ea94b86c7e.